project/deps.sc

```diff
@@ -140,7 +140,7 @@ object Deps {
   def munit                      = ivy"org.scalameta::munit:0.7.29"
   def nativeTestRunner           = ivy"org.scala-native::test-runner:${Versions.scalaNative}"
   def nativeTools                = ivy"org.scala-native::tools:${Versions.scalaNative}"
-  def osLib                      = ivy"com.lihaoyi::os-lib:0.9.1"
+  def osLib                      = ivy"com.lihaoyi::os-lib:0.9.2"
```

Comment:
This bump will be included in #2514, which I will merge as soon as the CI passes